core/templates/components/rubrics-editor/rubrics-editor.directive.html

```diff
@@ -8,7 +8,7 @@
         <div class="oppia-rule-preview">
           <div class="oppia-click-to-start-editing" ng-click="$ctrl.openExplanationEditor(rubric.getDifficulty(), $index)">
             <i ng-if="$ctrl.isEditable()"
-               class="material-icons oppia-editor-edit-icon float-right protractor-test-edit-rubric-explanation-<[rubricIndex]>"
+               class="material-icons oppia-editor-edit-icon float-right protractor-test-edit-rubric-explanation-<[rubric.getDifficulty()]>"
```

Comment:
Just to check ... I thought we were going to have multiple rubrics with the same difficulty. So this won't be unique in general? That seems like an issue...?

(Ditto elsewhere. Maybe you can find some other way to name them uniquely?)

Reply:
Yeah, all explanations for a difficulty will have the same name and they will be accessed using its index (using element.all on this class). 
Since explanations have no identifier, if they are to be uniquely identified here, the index has to be appended so we have to use indices to refer to them anyway, hence using element.all on this class, in the test, to access all explanations for a difficulty.

Reply:
OK, thanks! Maybe explain in the Protractor code.

Reply:
Done